failed to build kf5-kdesu 5.8, fatal error: 'X11/X.h' file not found

Disabling X11 probing fixes it. See https://github.com/vicamo/homebrew-kde-frameworks/commit/e8c6abf2733e486f8cd59b2488e180cb18b2489f for the patch.

```
Building CXX object src/kdesud/CMakeFiles/kdesud.dir/kdesud.cpp.o
cd /tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud && /usr/local/Library/ENV/4.3/clang++   -DKCOREADDONS_LIB -DQT_CORE_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_STRICT_ITERATORS -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -D_DARWIN_C_SOURCE -std=c++0x -fno-exceptions -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -I/tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud -I/tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src -I/opt/X11/include -isystem /usr/local/include/KF5/KPty -isystem /usr/local/include/KF5 -iframework /usr/local/opt/qt5/lib -isystem /usr/local/opt/qt5/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt5/mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5/KI18n    -o CMakeFiles/kdesud.dir/kdesud.cpp.o -c /tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud/kdesud.cpp
Building CXX object src/kdesud/CMakeFiles/kdesud.dir/kdesud_automoc.cpp.o
cd /tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud && /usr/local/Library/ENV/4.3/clang++   -DKCOREADDONS_LIB -DQT_CORE_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_STRICT_ITERATORS -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -D_DARWIN_C_SOURCE -std=c++0x -fno-exceptions -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -I/tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud -I/tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src -I/opt/X11/include -isystem /usr/local/include/KF5/KPty -isystem /usr/local/include/KF5 -iframework /usr/local/opt/qt5/lib -isystem /usr/local/opt/qt5/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt5/mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5/KI18n    -o CMakeFiles/kdesud.dir/kdesud_automoc.cpp.o -c /tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud/kdesud_automoc.cpp
/tmp/kf5-kdesu-X6xKk8/kdesu-5.8.0/src/kdesud/kdesud.cpp:77:10: fatal error: 'X11/X.h' file not found
#include <X11/X.h>
         ^
1 error generated.
make[2]: *** [src/kdesud/CMakeFiles/kdesud.dir/kdesud.cpp.o] Error 1
make[1]: *** [src/kdesud/CMakeFiles/kdesud.dir/all] Error 2
make: *** [all] Error 2
```

--- a/kf5-kdesu.rb
+++ b/kf5-kdesu.rb
@@ -13,6 +13,10 @@ class Kf5Kdesu < Formula
   depends_on "qt5" => "with-d-bus"
   depends_on "gettext" => :build
 
+  def patches
+    DATA
+  end
+
   def install
     args = std_cmake_args
     args << "-DCMAKE_CXX_FLAGS='-D_DARWIN_C_SOURCE'"
@@ -22,3 +26,19 @@ class Kf5Kdesu < Formula
     prefix.install "install_manifest.txt"
   end
 end
+
+__END__
+diff -Nu a/CMakeLists.txt a/CMakeLists.txt.new
+--- a/CMakeLists.txt	2015-03-07 22:41:59.000000000 +0800
++++ a/CMakeLists.txt.new	2015-03-14 23:08:17.000000000 +0800
+@@ -18,10 +18,6 @@
+ find_package(KF5Service ${KF5_DEP_VERSION} REQUIRED)
+ find_package(KF5Pty ${KF5_DEP_VERSION} REQUIRED)
+ 
+-#optional features
+-find_package(X11)
+-set(HAVE_X11 ${X11_FOUND})
+-
+ include(FeatureSummary)
+ include(GenerateExportHeader)
+ include(ECMSetupVersion)

